config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -319,7 +319,8 @@ presubmits:
 
   - name: pull-kubernetes-e2e-kind-ipv6
     optional: true
-    always_run: false # manual for testing new configurations
+    always_run: true
+    skip_report: true
```

Comment:
on second thought we shouldn't skip report, as we want to make this blocking sooner rather than later ...

I'll file a follow up given the time in spain :upside_down_face:

Reply:
nits fixed here: https://github.com/kubernetes/test-infra/pull/15801 (didn't touch this commit, it's a superset)